nifty-core/src/main/java/com/facebook/nifty/ssl/SSLClientConfiguration.java

```diff
@@ -0,0 +1,89 @@
+package com.facebook.nifty.ssl;
+
+import com.google.common.base.Throwables;
+import org.jboss.netty.handler.ssl.SslContext;
+import org.jboss.netty.handler.ssl.SslHandler;
+
+import javax.net.ssl.SSLException;
+import java.io.File;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+
+public class SSLClientConfiguration {
```

Comment:
The JVM does not have a consistent convention, but we have always treated abbreviations as words.  Without this you have problems when you have classes with multiple abbrebviations in a row like `EJBSSLConfig`.